L1Trigger/TrackTrigger/test/AnalyzerMC_cfg.py

```diff
@@ -0,0 +1,48 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
All jobs need a comment at the top explaining what they do. As we have several jobs analysing the tracking, it also needs to explain specifically what this one does.